Makefile

```diff
@@ -11,7 +11,12 @@ ADDLICENSE_CMD=go run github.com/google/addlicense
 ADDLICENCE_SCRIPT=${ADDLICENSE_CMD} -c "Coinbase, Inc." -l "apache" -v
 GOLINES_CMD=go run github.com/segmentio/golines
 GOVERALLS_CMD=go run github.com/mattn/goveralls
+COVERAGE_TEST_DIRECTORIES=./configuration/... ./internal/constructor/... \
+													./internal/logger/... ./internal/scenario/... \
```

Comment:
format?

Reply:
oh weird, doesn't look like that locally

Reply:
fixed!